.tekton/recert-4-20-pull-request.yaml

```
@@ -8,7 +8,28 @@ metadata:
+        'ouger/***'.pathChanged() ||
+        'src/***'.pathChanged() ||
+        'vendor/***'.pathChanged() ||
+        'build.rs'.pathChanged() ||
+        'Cargo.lock'.pathChanged() ||
+        'Cargo.toml'.pathChanged() ||
+        'ownership.sh'.pathChanged() ||
+        'reproto'.pathChanged() ||
+        'run_seed'.pathChanged() ||
+        'vendor.sh'.pathChanged() ||
+        '.tekton/recert-4-20-pull-request.yaml'.pathChanged() ||
+        '.konflux/Dockerfile'.pathChanged() || 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)